content/en/docs/collector/building/receiver.md

```diff
@@ -1383,7 +1383,8 @@ creating a trace.
 You will start with a type called `ptrace.ResourceSpans` which represents the
 resource and all the operations that it either originated or received while
 participating in a trace. You can find its definition within the
-[/pdata/internal/data/protogen/trace/v1/trace.pb.go](<https://github.com/open-telemetry/opentelemetry-collector/blob/v{{% param vers %}}/pdata/internal/data/protogen/trace/v1/trace.pb.go>).
+[/pdata/ptrace/generated_resourcespans.go](https://github.com/open-telemetry/opentelemetry-collector/blob/v0.139.0/pdata/ptrace/generated_resourcespans.go)
```

Comment:
@tiffany76 - do you have a note somewhere to remember to replace the hard-coded version by `v{{% param vers %}}` once again?

PR #8328 is probably the right place to do it. WDYT?